src/main/java/mate/academy/spring/service/mapper/MovieSessionDtoMapper.java

```diff
@@ -0,0 +1,36 @@
+package mate.academy.spring.service.mapper;
+
+import mate.academy.spring.model.MovieSession;
+import mate.academy.spring.model.dto.MovieSessionRequestDto;
+import mate.academy.spring.model.dto.MovieSessionResponseDto;
+import mate.academy.spring.service.CinemaHallService;
+import mate.academy.spring.service.MovieService;
+import org.springframework.stereotype.Component;
+
+@Component
+public class MovieSessionDtoMapper {
+    private final MovieService movieService;
+    private final CinemaHallService cinemaHallService;
+
+    public MovieSessionDtoMapper(MovieService movieService, CinemaHallService cinemaHallService) {
```

Comment:
If you used it in other constructors, use it here too
```suggestion
    @Autowired
    public MovieSessionDtoMapper(MovieService movieService, CinemaHallService cinemaHallService) {
```